docs/precommit.md

```diff
@@ -61,7 +61,28 @@ Copy the following config into your `.pre-commit-config.yaml` file:
 
 Read more at [mirror of prettier package for pre-commit](https://github.com/pre-commit/mirrors-prettier) and the [pre-commit](https://pre-commit.com) website.
 
-## Option 4. [git-format-staged](https://github.com/hallettj/git-format-staged)
+## Option 4. [Husky.Net](https://github.com/alirezanet/Husky.Net)
+
+**Use Case:** A dotnet solution to use Prettier along with other code quality tools (e.g. dotnet-format, ESLint, Stylelint, etc.). It supports multiple file states (staged - last-commit, git-files etc.)
+
+```bash
+dotnet tool install husky
+dotnet husky install
+dotnet husky add .husky/pre-commit
+```
+
+after installation you can add prettier task to the `task-runner.json`.
+
+```json
+{
+   "command": "npx",
+   "args": ["prettier", "--write", "${staged}"],
```

Comment:
We suggest use `--ignore-unknown` flag and include all files.

Reply:
good suggestion thanks. updated.